loopd/daemon.go

```diff
@@ -85,7 +112,7 @@ func daemon(config *Config, lisCfg *listenerCfg) error {
 	// Instantiate the loopd gRPC server.
 	server := swapClientServer{
 		impl:        swapClient,
-		lnd:         &lnd.LndServices,
+		lnd:         &d.lnd.LndServices,
```

Comment:
Didn't `New` already create this `swapClientServer`?

Reply:
Ok, I see changes in the next commit. Commit structure can probably be optimized

Reply:
Ah yes, I missed that duplicated initialization of `swapClientServer`. Fixed the commit.